proxy/src/main/resources/translations/de_de.json

```diff
@@ -0,0 +1,195 @@
+{
+  "velocity.only-online-mode": {
+    "text": "Dieser Server akzeptiert nur Verbindungen von Spielern im Online-Modus.",
+    "color": "red",
+    "extra": [
+      {
+        "text": "\n\nHast Du deinen Benutzernamen geändert? Melde dich bei Minecraft ab und an und versuche es erneut.",
+        "color": "gray"
+      }
+    ]
+  },
+  "velocity.no-proxy-behind-proxy": {
+    "text": "Velocity hinter Velocity zu betreiben wird nicht unterstüzt.",
+    "color": "red"
+  },
+  "velocity.no-available-servers": {
+    "text": "Keine verfügbaren Server.",
+    "color": "red"
+  },
+  "velocity.already-connected": {
+    "text": "Du bist bereits mit diesem Proxy verbunden!",
+    "color": "red"
+  },
+  "velocity.moved-to-new-server": {
+    "text": "Du wurdest von dem Server auf dem Du warst gekickt.",
```

Comment:
There are commas missing: 'Du wurdest von dem Server, auf dem Du warst, gekickt.'
Furthermore, I would suggest to rather use this translation: 'Du wurdest von dem Server, auf dem Du vorher warst, geworfen.'

Reply:
Yours sounds a bit better, thanks +1